app/serializers/auction_with_bid_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class AuctionWithBidSerializer
+  def self.serialize(auction_with_bid)
+    auction = auction_with_bid.auction.attributes
+
+    if auction_with_bid.bid
+      auction.merge(bid: auction_with_bid.bid.attributes)
```

Comment:
Filter out user id so it's not exposed
Filter out created_at, updated_at

only display price

Reply:
@ijpiantanida I was able to get the last bid added to the query, but it includes all the data. I can tell the Auction serializer that it `has_many :bids`, and that will automatically make auction queries include the associated bids and serializes them, but it returns ALL the bids instead of just the last one. I'm sure here's another way, but I'm not sure how.